proto/p4/config/v1/p4info.proto

```diff
@@ -217,8 +217,16 @@ message Table {
   }
   // is idle timeout supported for this table?
   IdleTimeoutBehavior idle_timeout_behavior = 9;
-  // table with static P4 entries, cannot be modified at runtime
+  // is_const_table is true if and only if the table's entries are
```

Comment:
Optional: Perhaps just start the comment with "True if and only if ..." since the subject is implied? But that's just my personal preference and you may prefer it this way.

Ditto for the other new fields.

Reply:
I believe this comment is addressed in or before commit 17